pkgs/top-level/ocaml-packages.nix

```diff
@@ -55,6 +55,8 @@ let
 
     base64 = callPackage ../development/ocaml-modules/base64 { };
 
+    base64_3_1_0 = callPackage ../development/ocaml-modules/base64/v3_1_0.nix { };
```

Comment:
This attribute is too specific. `base64_3` should be enough.

Also, I would rather like a specific name for the legacy version (`base64_2`).

Reply:
I shuffled that around (base64_2 and the default is now version 3)